sdk/spring/spring-messaging-azure-storage-queue/src/main/java/com/azure/spring/storage/queue/core/properties/StorageQueueProperties.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.storage.queue.core.properties;
+
+import com.azure.spring.core.properties.AzureHttpSdkProperties;
+import com.azure.spring.service.storage.queue.QueueServiceClientProperties;
+import com.azure.storage.queue.QueueMessageEncoding;
+import com.azure.storage.queue.QueueServiceVersion;
+
+/**
+ * Properties for Azure Storage Queue service.
+ */
+public class StorageQueueProperties extends AzureHttpSdkProperties implements QueueServiceClientProperties {
```

Comment:
Do we already have one in the service module?

Reply:
we have an interface of `QueueServiceClientProperties` in service module but not a POJO class.